model-optimizer/automation/package_BOM.txt

```diff
@@ -1101,4 +1101,3 @@ requirements_kaldi.txt
 requirements_mxnet.txt
 requirements_onnx.txt
 requirements_tf.txt
-requirements_tf2.txt
```

Comment:
Quick grep for "requirements_tf2" shows several places in the documentation where it is mentioned:

```
bash-3.2$ grep -Isrn requirements_tf2 ./
.//MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md:339:In order to convert TensorFlow* 2 models, installation of dependencies from `requirements_tf2.txt` is required.
.//MO_DG/prepare_model/Config_Model_Optimizer.md:139:pip3 install -r requirements_tf2.txt
```

Reply:
Now it's removed